packages/frontend/src/components/Election/Results/Results.tsx

```diff
@@ -507,6 +509,8 @@ export default function Results({ race, results }: {race: Race, results: Electio
     .map((s,i) => ([<React.Fragment key={i*2}>{s}</React.Fragment>, <React.Fragment key={i*2+1}>&nbsp;</React.Fragment>]))
     .flat()
 
+  /* console.log('votingMethod', results.votingMethod); */
```

Comment:
not a blocker, but for future PRs can you remove the commented console log messages?